lib/logasm.js

```diff
@@ -75,15 +75,15 @@ class Logasm {
       let data = {};
 
       if (metadata instanceof Error) {
-        data['error'] = metadata.toString();
-        if (metadata.stack) { data['error.stack_trace'] = metadata.stack; }
+        data['error'] = { message: metadata.toString() };
+        if (metadata.stack) { data['error']['stack'] = metadata.stack; };
```

Comment:
do we need to check null value? Most errors should have stack and does it cause any problems if it is set to null?

Reply:
I don't know if all errors have `stack`. Locally it would just print `stack=undefined` if stack is missing. I'd rather not remove that check because there must have been a reason for the check.

Reply:
The check has been present from the first commit https://github.com/salemove/node-logasm/commit/cf1e0fb8fb30f0d531becca7ca048675bd7ea8c1#diff-eaa1681d1963633f8e1135dd83237d17R120